### DISCUSSION TITLE
Ignore Apps? Or Ignore Hotkeys When Certain App Has Focus?

### DISCUSSION THREAD

**Discussion by Stickyhavr** [rxhanson/RectanglePro-Community#55] [Q&A]
Hello,
I love that Rectangle Pro is always listening, but it would be nice to be able to ignore certain hotkeys when certain apps have the focus. Is there any way to ignore an app?

Specifically, when I am audio editing my hotkeys for Window Throw Move and Resize often get triggered accidentally.  I suppose I could quite Rectangle Pro temporarily, but then I might forget to start it back up again and then be annoyed/confused when it wasn't working the next time I needed it.

To make things more complicated, sometimes I do use Rectangle Pro to move windows around in my audio editor, so it would be ideal if I could just ignore the Window Throw actions, although I understand that might be too specific a use case to be useful for others. Thanks!

**Comment by rxhanson** [comment]
With the app you want to ignore frontmost, open the menu and select "Ignore [app]", below all of the window actions.

> To make things more complicated, sometimes I do use Rectangle Pro to move windows around in my audio editor, so it would be ideal if I could just ignore the Window Throw actions, although I understand that might be too specific a use case to be useful for others. 

I wouldn't add something like that to the UI, but can see adding it as a terminal command config if there's enough demand.

**Comment by Stickyhavr** [comment]
Oh, sorry about that.  A Google search would have answered my question. I see it now on the main rectangle GitHub https://github.com/rxhanson/Rectangle and also in this answer https://github.com/rxhanson/Rectangle/issues/47

I did search through answers before I posted but only here and not on the main rectangle page, oops. Hopefully this thread will help someone else down the road. I don't use the menu bar icon, so I didn't think to look there. I just went through all of the preferences and when I couldn't find it, I posted. Again, my apologies. Thanks for your answer.

Ignoring will be fine for the time being. But I will set a reminder to check the terminal commands every once in a while, just to see what's new. Is this the most up-to-date list for those: https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md ?

Thanks again!
